card/mod/history/lib/card/change.rb

```diff
@@ -1,9 +1,29 @@
 # -*- encoding : utf-8 -*-
 class Card
+  # A _change_ is an alteration to a card's name, type, content, or trash state.
+  # Together, {Act acts}, {Action actions}, and {Change changes} comprise a
+  # comprehensive {Card card} history tracking system.
+  #
+  # For example, if a given web submission changes both the name and type of
+  # card, that would be recorded as one {Action action} with two
+  # {Change changes}.
+  #
+  # A {change} records:
+  #
+  # * the _field_ changed
+  # * the new _value_ of that field
+  # * the {Action action} of which the change is part
+  #
   class Change < ActiveRecord::Base
-    belongs_to :action, foreign_key: :card_action_id, inverse_of: :card_changes
+    belongs_to :action, foreign_key: :card_action_id,
+                        inverse_of: :card_changes
+
+    # lists the database fields for which changes are recorded
+    TRACKED_FIELDS = %w(name type_id db_content trash).freeze
```

Comment:
maybe.  there are things like this that I don't know how to replace:
```
 Card::Change::TRACKED_FIELDS.each_with_object
```
...because I don't know how to get access to the original array.  but maybe I could do `enum :field, TRACKED_FIELDS`  or something?